lib/config/config-file.js

```diff
@@ -313,7 +313,7 @@ function write(config, filePath) {
 function getBaseDir(configFilePath) {
 
     // calculates the path of the project including ESLint as dependency
-    var projectPath = path.resolve(__dirname, "../../../");
+    var projectPath = path.resolve(__dirname, "../../../../");
```

Comment:
I'm wondering that [this parser path](https://github.com/eslint/eslint/blob/master/lib/config/config-file.js#L515-L521) is working fine with this wrong `basedir` and `lookupPath`?